visualizations/store-map-viz/components/Regions.tsx

```diff
@@ -94,6 +132,10 @@ const Regions = () => {
               location={location}
               tooltipConfig={tooltipConfig}
               defaultHeader={feature.properties.ADMIN}
+              heatMap={getGradientColor}
+              heatMapSteps={gradientSteps}
```

Comment:
Theres probbaly a better way to do this. In order for the region color to update when the config changed i had to pass ll this information over to use in the useEffect trigger. I cant help but feel the gradient generator function should live within Region.tsx, but its here because it needs information form all the regions for the gradient bucketing to be derived.